meta-lmp-base/recipes-support/lmp-device-auto-register/lmp-device-auto-register/lmp-device-auto-register

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh -e
 
 TOKEN_FILE=${TOKEN_FILE-/etc/lmp-device-register-token}
+TAGS=""
+
+if [ -f /etc/sota/tags ] ; then
+       # tags file should be formatted properly already
+       # proper format is tag1,tag2,tag3
```

Comment:
I *think* we only allow a single tag. The plural form of our wording is historical baggage

Reply:
let me check, I used only one so far.